.github/workflows/site-checker.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will check to see if if sites in the webring are reachable
+
+name: Reachable Sites Check
+
+on:
+  push:
+    branches: [ master ]
+  schedule:
+    - cron:  '0 0/4 * * *' #every 4 hours
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js 14.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 14.x
+    - name: install deps
+      run: (cd .github/site-checker && npm i)
```

Comment:
Might want to use `npm ci` so that the lock file is unconditionally used, I think? Or is that the default now?

Reply:
updated ✅